README.md

```diff
@@ -69,7 +69,7 @@ We use localstack and opensearch, and run them via Docker.
 
 ```bash
 docker run -dp 3042:8080 -p 3040:4566 -p 3041:4566 localstack/localstack
-docker run -dp 9200:9200 -e "discovery.type=single-node" opensearchproject/opensearch:1.1.0
+docker run -dp 9200:9200 -e "discovery.type=single-node" opensearchproject/opensearch:2.9.0
```

Comment:
Maybe we save these versions somewhere so that they're consistent in dev + ci?

Reply:
Done in https://github.com/exercism/website/pull/5824/commits/57ac597e9de284a808ec2009971ad1a7a4290590

Looks to be working: https://github.com/exercism/website/actions/runs/5724277008/job/15510467895